airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeDestination.java

```diff
@@ -39,10 +39,10 @@ public SerializedAirbyteMessageConsumer getSerializedMessageConsumer(final JsonN
       throws Exception {
     log.info("destination class: {}", getClass());
     // detect if running on internal staging for snowflake, if so run consumer2.
-    final boolean useAsyncSnowflake = false;
-    // final boolean useAsyncSnowflake = config.has("loading_method")
-    // && config.get("loading_method").has("method")
-    // && config.get("loading_method").get("method").asText().equals("Internal Staging");
+    // final boolean useAsyncSnowflake = false;
+    final boolean useAsyncSnowflake = config.has("loading_method")
+        && config.get("loading_method").has("method")
+        && config.get("loading_method").get("method").asText().equals("Internal Staging");
```

Comment:
this should be commented out before we merge everything in this branch to master

Reply:
yup. done